src/Content/Text/BBCode.php

```diff
@@ -1361,6 +1361,15 @@ function ($match) {
 				. '</a>';
 		}, $text);
 
+		// We need no target="_blank" for local links
+		// convert links start with System::baseUrl() as local link
+		$escapedBaseUrl = str_replace('://', '\:\/\/', System::baseUrl());
```

Comment:
Please use `preg_escape()` instead.

Reply:
Sorry, `preg_quote`.

Reply:
Ah great. Didnt know that function. I always did it "manually"
done